cpu/atmega_common/pm/pm.c

```diff
@@ -26,6 +26,14 @@
 #include "irq.h"
 #include "periph/pm.h"
 
+void pm_set_lowest(void) {}
```

Comment:
shouldn't this be in the common pm_fallback_set_lowest?

Reply:
ACK